src/solve.jl

```diff
@@ -1,4 +1,4 @@
-function solve(prob::AbstractSteadyStateProblem,alg::SteadyStateDiffEqAlgorithm,args...;kwargs...)
+function solve(prob::AbstractSteadyStateProblem,alg::SteadyStateDiffEqAlgorithm,abstol=1e-8,args...;kwargs...)
```

Comment:
this should be a keyword argument instead of an optional argument, and that's what's causing the test failure. It's a keyword argument if it goes after the `;`

Reply:
I have change the order in the new commit.